change/@uifabric-experiments-2019-09-05-16-38-24-jahnp-nuke-fluent-theme.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
Might not be important but I think this actually has 0 functional effect on experiments, so can be `none`

Reply:
Ahh, very good callout—thanks @cliffkoh . Will adjust.